### ISSUE TITLE
Files don't load on latest forge version.

### ISSUE THREAD

**Issue by goreae** [jaredlll08/JSON-ables#13]
I'm working on making tool materials for the metals in Advent of Ascension. I got limonite mostly working, but needed to update forge to the latest version. When I did, my file refused to load, and the example file wouldn't load either. Backdating the forge version to 1352 fixes it and allows all of the files to load. In case I'm an absolute idiot and screwed up my JSON somewhere, here it is:http://paste.feed-the-beast.com/view/2f5c4d9c

**Comment by jaredlll08** [comment]
uh that is the template.json...

**Comment by goreae** [comment]
Yes it is. I was using it to be sure that the problem wasn't something I was doing wrong. That works perfectly fine on earlier forge versions and later versions without the fluid section, but doesn't work at all with the custom fluid on later forge versions.
